### ISSUE TITLE
directory "/blah" is not using a known version control system

### ISSUE THREAD

**Issue by davidscholberg** [go-bootstrap/go-bootstrap#41]
I just updated to the latest version (250fa9c4c6db3c0e91fe8d163f535583dcd25156), and I get this error when trying to create a new project:

```
$ go-bootstrap -dir github.com/davidscholberg/go-bootstrap-test -template core
2016/02/28 14:39:24 Creating /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test...
2016/02/28 14:39:24 Copying project template directory to /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test...
2016/02/28 14:39:24 Replacing placeholder variables on davidscholberg/go-bootstrap-test...
2016/02/28 14:39:24 Running go get -u ./...
2016/02/28 14:39:24 package github.com/davidscholberg/go-bootstrap-test: directory "/home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test" is not using a known version control system
package github.com/davidscholberg/go-bootstrap-test/application: directory "/home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/application" is not using a known version control system
package github.com/davidscholberg/go-bootstrap-test/handlers: directory "/home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/handlers" is not using a known version control system
package github.com/davidscholberg/go-bootstrap-test/libhttp: directory "/home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/libhttp" is not using a known version control system
package github.com/davidscholberg/go-bootstrap-test/libstring: directory "/home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/libstring" is not using a known version control system
package github.com/davidscholberg/go-bootstrap-test/middlewares: directory "/home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/middlewares" is not using a known version control system

exit status 1
```

**Comment by davidscholberg** [comment]
FYI, here's my go version:

```
$ go version
go version go1.6 linux/amd64
```

**Comment by davidscholberg** [comment]
I reverted go-bootstrap to a47291855be16baec5f83002ce2a60cedb03a3c7 and got a little farther, but received another error:

```
$ go-bootstrap -dir github.com/davidscholberg/go-bootstrap-test -template core                                                                                                        [928/1790]
2016/02/28 14:55:19 Creating /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test...
2016/02/28 14:55:19 Copying project template directory to /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test...
2016/02/28 14:55:19 Replacing placeholder variables on davidscholberg/go-bootstrap-test...
2016/02/28 14:55:19 Running go get ./...
2016/02/28 14:55:21 Installing github.com/tools/godep...
2016/02/28 14:55:21 Running git init...
2016/02/28 14:55:21 Running godep save ./...
2016/02/28 14:55:22 Running godep go test ./...
2016/02/28 14:55:22 vendor/github.com/Sirupsen/logrus/examples/hook/hook.go:5:2: cannot find package "gopkg.in/gemnasium/logrus-airbrake-hook.v2" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/gopkg.in/gemnasium/logrus-airbrake-hook.v2 (vendor tree)
        /usr/lib/go/src/gopkg.in/gemnasium/logrus-airbrake-hook.v2 (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/gopkg.in/gemnasium/logrus-airbrake-hook.v2 (from $GOPATH)
        /home/david/src/go/src/gopkg.in/gemnasium/logrus-airbrake-hook.v2
vendor/github.com/carbocation/interpose/adaptors/negroni.go:6:2: cannot find package "github.com/codegangsta/negroni" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/codegangsta/negroni (vendor tree)
        /usr/lib/go/src/github.com/codegangsta/negroni (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/codegangsta/negroni (from $GOPATH)
        /home/david/src/go/src/github.com/codegangsta/negroni
vendor/github.com/carbocation/interpose/adaptors/martini.go:6:2: cannot find package "github.com/go-martini/martini" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/go-martini/martini (vendor tree)
        /usr/lib/go/src/github.com/go-martini/martini (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/go-martini/martini (from $GOPATH)
        /home/david/src/go/src/github.com/go-martini/martini
vendor/github.com/carbocation/interpose/examples/adaptors/logrus/main.go:12:2: cannot find package "github.com/meatballhat/negroni-logrus" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/meatballhat/negroni-logrus (vendor tree)
        /usr/lib/go/src/github.com/meatballhat/negroni-logrus (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/meatballhat/negroni-logrus (from $GOPATH)
        /home/david/src/go/src/github.com/meatballhat/negroni-logrus
vendor/github.com/carbocation/interpose/examples/adaptors/logrus/main.go:13:2: cannot find package "github.com/stretchr/graceful" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/stretchr/graceful (vendor tree)
        /usr/lib/go/src/github.com/stretchr/graceful (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/stretchr/graceful (from $GOPATH)
        /home/david/src/go/src/github.com/stretchr/graceful
vendor/github.com/carbocation/interpose/examples/adaptors/martiniauth/main.go:10:2: cannot find package "github.com/martini-contrib/auth" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/martini-contrib/auth (vendor tree)
        /usr/lib/go/src/github.com/martini-contrib/auth (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/martini-contrib/auth (from $GOPATH)
        /home/david/src/go/src/github.com/martini-contrib/auth
vendor/github.com/carbocation/interpose/middleware/gorillalog.go:7:2: cannot find package "github.com/carbocation/handlers" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/carbocation/handlers (vendor tree)
        /usr/lib/go/src/github.com/carbocation/handlers (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/carbocation/handlers (from $GOPATH)
        /home/david/src/go/src/github.com/carbocation/handlers
vendor/github.com/carbocation/interpose/middleware/buffer.go:6:2: cannot find package "github.com/goods/httpbuf" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/goods/httpbuf (vendor tree)
        /usr/lib/go/src/github.com/goods/httpbuf (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/goods/httpbuf (from $GOPATH)
        /home/david/src/go/src/github.com/goods/httpbuf
vendor/github.com/carbocation/interpose/middleware/nosurf.go:6:2: cannot find package "github.com/justinas/nosurf" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/justinas/nosurf (vendor tree)
        /usr/lib/go/src/github.com/justinas/nosurf (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/justinas/nosurf (from $GOPATH)
        /home/david/src/go/src/github.com/justinas/nosurf
vendor/github.com/carbocation/interpose/middleware/negronigzip.go:7:2: cannot find package "github.com/phyber/negroni-gzip/gzip" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/phyber/negroni-gzip/gzip (vendor tree)
        /usr/lib/go/src/github.com/phyber/negroni-gzip/gzip (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/phyber/negroni-gzip/gzip (from $GOPATH)
        /home/david/src/go/src/github.com/phyber/negroni-gzip/gzip
vendor/github.com/carbocation/interpose/examples/secure/main.go:11:2: cannot find package "github.com/unrolled/secure" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/unrolled/secure (vendor tree)
        /usr/lib/go/src/github.com/unrolled/secure (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/unrolled/secure (from $GOPATH)
        /home/david/src/go/src/github.com/unrolled/secure
vendor/github.com/kr/text/mc/mc.go:13:2: cannot find package "github.com/kr/pty" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/kr/pty (vendor tree)
        /usr/lib/go/src/github.com/kr/pty (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/kr/pty (from $GOPATH)
        /home/david/src/go/src/github.com/kr/pty
vendor/github.com/spf13/viper/remote/remote.go:12:2: cannot find package "github.com/xordataexchange/crypt/config" in any of:
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/vendor/github.com/xordataexchange/crypt/config (vendor tree)
        /usr/lib/go/src/github.com/xordataexchange/crypt/config (from $GOROOT)
        /home/david/src/go/src/github.com/davidscholberg/go-bootstrap-test/Godeps/_workspace/src/github.com/xordataexchange/crypt/config (from $GOPATH)
        /home/david/src/go/src/github.com/xordataexchange/crypt/config
godep: go exit status 1
```

I was able to solve that error by setting the GO15VENDOREXPERIMENT environment variable to 0. So it looks like something is not working with the new vendor directory feature in Go 1.6.

This is, of course, a separate issue from the one I posted above, so I'll create a new issue for it if you want.

**Comment by didip** [comment]
It does look like 2 different issues, but let's just solve them all here.

1. Your first problem is a legit bug, caused by a recent change: `go get -u ./...`. I have reverted that change.

2. It definitely looks like Godeps have some bugs transitioning to the new `/vendor` layout. I have decided to remove godeps for now since the community may switch to a different vendoring tool in the future.

Fixes: https://github.com/go-bootstrap/go-bootstrap/commit/6c2e50f0ae0ec0fd376b9cf49028eb0e0cca8555
